release-notes/azure/v11.1.0.md

```diff
@@ -0,0 +1,46 @@
+## :zap: Giant Swarm Release 11.1.0 for Azure is now active for you! :zap:
+
+This release introduces support for multiple Availability Zones, as well as graceful upgrades, and improved worker labeling.
```

Comment:
@tuommaki can you provide more details about graceful upgrades? I mean what will customers notice - shorter upgrade time and no workload disruption (if they have at least 2 worker nodes and this funny-named thingy (PDB???) set up) I think it would be nice to explain this here.